README.md

```diff
@@ -62,24 +73,34 @@ RxC Voice - http://localhost:3000
 
 ## Creating users and accessing the site for testing
 
-1. Create a superuser to access the admin site
+1. Run migrations
+
+```
+docker exec -it rxc-voice_api_1 python manage.py migrate
```

Comment:
migrations are run automatically on startup-- see rxc-voice/scripts/deploy/api_deploy_dev.sh

Reply:
I was prompted to do it manually and couldn't run the app until i did so